modules/test/playwright/tests/journal-web/journalPage.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {journalPagesTest} from './fixtures/journalPagesTest';
+import getDataStructureDefinition from './utils/getDataStructureDefinition';
+import getBasicWebContentStructureId from '../../utils/structured-content/getBasicWebContentStructureId';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	isolatedSiteTest,
+	journalPagesTest,
+	loginTest()
+);
+
+test('LPP-50468 After clicking on Clear (filter by structrure) you can see all the web contents', async ({
+	apiHelpers,
+	journalPage,
+	page,
+	site,
+}) => {
+	//1) create basic web content
+	const basicWebContentStructureId = await getBasicWebContentStructureId(apiHelpers);
+
+	await apiHelpers.jsonWebServicesJournal.addWebContent({
+		ddmStructureId: basicWebContentStructureId,
+		groupId: site.id,
+		titleMap: {en_US: 'First Web content'}
+	});
+
+	//2) Create structure
+	const dataDefinition = getDataStructureDefinition({
+		defaultLanguageId: 'en_US',
+		fields: [{name: 'Text', repeatable: false}],
+		name: 'Structure Test',
+	});
+
+	const structure = await apiHelpers.dataEngine.createStructure(site.id, dataDefinition);
+
+	//3) Create web content based on structure
+	await apiHelpers.jsonWebServicesJournal.addWebContent({
+		ddmStructureId: structure.id,
+		groupId: site.id,
+		titleMap: {en_US: 'Second Web Content'}
+	});
```

Comment:
I suggest moving into `/utils/` function and avoiding comments. 

Something like [blogs-web/blogEntry.spec.ts](https://github.com/liferay/liferay-portal/blob/d66cc464ba0724b676b62463bd2b2a28a6d9b402/modules/test/playwright/tests/blogs-web/blogEntry.spec.ts#L81-L89) and [blogs-web/utils/](https://github.com/liferay/liferay-portal/tree/master/modules/test/playwright/tests/blogs-web/utils/).

But I think you can do it all in one file ❤️